crates/egui/src/data/input.rs

```diff
@@ -336,7 +336,7 @@ pub struct DroppedFile {
     pub last_modified: Option<std::time::SystemTime>,
 
     /// Set by the `eframe` web backend.
-    pub bytes: Option<std::sync::Arc<[u8]>>,
+    pub stream_url: Option<String>,
```

Comment:
How is this supposed to be used?

Reply:
this create a url

Then the user can fetch() the file

Maybe i should provide a helper function to retreive the data ?

Reply:
And please make sure the `egui_example_app` demonstrates how to download the file from this URL

Reply:
A helper function would be great yes.

Reply:
![image](https://github.com/emilk/egui/assets/56606507/6618886f-9177-4fc1-8f84-45b7527137da)